lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx

```diff
@@ -257,6 +258,7 @@ const StandardUserInterface = observer(
             viewState={this.props.viewState}
           />
           <div className={Styles.storyWrapper}>
+            <InternetExplorerOverlay viewState={this.props.viewState} />
```

Comment:
Simply don't mount this at all for next

Reply:
Oops I already merged this, I'll create a one-line PR that removes this line from `next`